test/functional/harness/test-runner.js

```diff
@@ -0,0 +1,96 @@
+var testCafe     = require('./index').testCafe;
+var browsersInfo = require('./index').browsersInfo;

```

Comment:
`test-runner` requires `harness`. Then, `test-runner` is not a part of harness, so move it up to `test/functional/test-runner.js`